__test__/add_remove.test.js

```diff
@@ -0,0 +1,20 @@
+import { addElement, deletetask } from '../src/add_remove.js';
```

Comment:
Please make sure you test most, if not all, functions involved in adding or deleting tasks. You only tested one for each.

<img width="451" alt="Screen Shot 2022-07-13 at 18 44 14" src="https://user-images.githubusercontent.com/82426682/178855398-a143d18e-7085-44c5-afc6-831fe4c80119.png">

Reply:
Dear, look at the requirement No.4, i.e.: 
**Take a look at each of these functions. Are they pure functions? If the answer is "yes" then writing tests for them should be straightforward.**
So, the addition and deletion are the only pure functions, functions related to these functions are not pure?